www/templates/control/main-control.html

```diff
@@ -1,18 +1,32 @@
 <ion-view view-title="{{'control.profile' | i18next}}" ng-class="ion-view-background">
   <ion-content>
-    <div class="control-list-item">
+    
+    <!-- opcode and logout - this row has two clicks! (and an argument for one of them) -->
+    <!-- <div class="control-list-item">
       <div ng-click="copyToClipboard(settings.auth.opcode)" class="control-list-text"><pre>{{settings.auth.opcode}}</pre></div>
       <div ng-click="logOut()" id ="switch-user" class="control-icon-button"><i class="icon ion-log-out"></i></div>
-    </div>
+    </div> -->
+    <setting-row text-key="settings.auth.opcode" icon-name="'clipboard'" action = "copyOpToClipboard"></setting-row>
+    <setting-row text-key="settings.auth.opcode" icon-name="'exit'" action = "logOut"></setting-row>
+
+    <!-- custom piece already? -->
     <enketo-demographics-button></enketo-demographics-button>
```

Comment:
Yes, this one is its own directive because it handles some more complex logic to load the survey.
I think it should be refactored before we try to convert it, so I would leave it as-is for right now